todo-app/src/components/AddingContainer.js

```diff
@@ -22,17 +18,21 @@ export default class AddingContainer extends Component {
     handleCreating = () => {
         let descriptionInput = this.state.description
```

Comment:
You should be using `const` here, not `let`. I think we've encountered this problem before. You should read [some articles](http://wesbos.com/let-vs-const/) about the difference so you get a good idea why I recommend this.

Also, you could save some time by pulling your variables out as we did in class:

    const { description, deadLine } = this.state